content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -467,6 +467,41 @@ Other alternatives for enforcing policies are being developed in the Kubernetes
 - [Kyverno](https://kyverno.io/policies/pod-security/)
 - [OPA Gatekeeper](https://github.com/open-policy-agent/gatekeeper)
 
+## Pod OS field
+
+Windows in Kubernetes has some limitations and differentiators from standard Linux-based
+workloads. Specifically, many of the Pod SecurityContext fields
+[have no effect on Windows](/docs/concepts/windows/intro/#compatibility-v1-pod-spec-containers-securitycontext).
+
+### Pod API validation changes
+For Kubernetes v{{< skew currentVersion >}} it is good practice to specify
+the `.spec.os` field for every Pod. The `.spec.os` field is available in Kubernetes
+v1.23 and later but was behind a
+[feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/)
+for Kubernetes v1.23 and v1.24.
+If you set the `pod.spec.os.name` to `linux`, the enforced security controls should be pertinent to Linux.
+Setting the `pod.spec.os.name` field to `windows` means that you don't need to apply Linux-specific restrictions 
+and the API validation for the pod object would fail if you apply Linux-specific restrictions to such pods.
+For example, a Windows Pod does not need to explicitly drop all capabilities for each container, even if 
+you are applying the `restricted` Pod security standard. If you don't specify the `pod.spec.os.name`, 
+Kubernetes takes a precautionary approach and applies all the Pod security measures that could be relevant 
+to Linux. 
+
+### Restricted Pod Security Standard changes
+Another important change starting Kubernetes v{{ < skew currentVersion >}} is that the pod restricted standard
+has been updated to use the `pod.spec.os.name` field. Based on the OS name, certain policies that are specific
+to a particualr OS can be relaxed for the other OS. 
+
+
+#### Policies specific to Linux
+- Privilege Escalation
+- Seccomp
+- Linux Capabilities
+
+The use of the HostProcess flag to create a HostProcess pod should only be done in alignment with the privileged policy.
+Creation of a Windows HostProcess pod is blocked under the baseline and restricted policies,
+so any HostProcess pod should be considered privileged.
```

Comment:
I think we can drop this. It's already covered in the policy table, and doesn't behave any differently with pod os set.